### DISCUSSION TITLE
When I re-opened my .hipnc file, I can not preview the cooked images.

### DISCUSSION THREAD

**Discussion by supwang** [stassius/StableHoudini#18] [Q&A] [answered]
Hi,

When I re-opened my .hipnc file, I can not preview the cooked images. There is only the icon of img in Composite View.
How to active the image preview?
Thank you.
<img width="479" alt="1" src="https://user-images.githubusercontent.com/4122964/235823575-24cc3cae-6ebf-4813-adce-ef21f4fa9e5c.png">

**Comment by stassius** [comment, marked as answer]
Image preview node only works with tasks. In order to use it you have to cook the upstream tasks. It will regenerate your images, so easier way would be to create a File Pattern node, point it to the target folder with `*.png` instead of a filename, and append the image_preview node afterwards. That way you can quickly preview all images in your folder without regenerating them.

**Reply by supwang** [reply]
Thank you.
I cooked the task, but there is still no image in Composite View. If I create a new file and follow your tutorial, the image can be shown in Composite View. But if I close the Houdini, then re-open the project and cooked the task. Nothing will be shown in Compsite View.
<img width="1649" alt="1" src="https://user-images.githubusercontent.com/4122964/236184994-48ee4a6a-785e-4866-b996-a69824d83bd2.png">

**Comment by supwang** [comment]
Solved.
If I use shortcut Shift+G, there has no image be shown in Composite View. But if I click the button "Cook", image can be shown in Composite View.
